packages/schemas/src/foundations/jsonb-types/hooks.ts

```diff
@@ -29,3 +66,16 @@ export const hookConfigGuard = z.object({
 });
 
 export type HookConfig = z.infer<typeof hookConfigGuard>;
+
+type ApiMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH';
+type ManagementApiHookKey = `${ApiMethod} ${string}`;
```

Comment:
this name is a little bit confusing since i'm thinking about the real keys (`keyof managementApiHooksRegistration`) when i see it. will it be better to inline this type below? i.e. `satisfies Record<`${ApiMethod} ${string}`, DataHookEvent>`